src/js/serial_backend.js

```diff
@@ -193,6 +193,12 @@ function finishClose(finishedCallback) {
     // reset active sensor indicators
     sensor_status(0);
 
+    // reset expert mode
+    ConfigStorage.get('permanentExpertMode', function (result) {
+        const checked = result.permanentExpertMode;
+        $('input[name="expertModeCheckbox"]').prop('checked', checked).change();
```

Comment:
We should use the new syntax:

```suggestion
        $('input[name="expertModeCheckbox"]').prop('checked', checked).trigger('change');
```

Reply:
Yea, didn't know that
Done :+1: